<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(209, 209, 209, 80);</string>
  </property>
  <widget class="QTextEdit" name="textEdit_os">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>344</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>354</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>操作系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>处理器型号</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_cpu">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>内存</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_mem">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_ip">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>40</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_ym">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>子网掩码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>默认网关</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_gate">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>130</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_mac">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>180</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>物理地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_copall">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开MIS电脑台账</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_quit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>451</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>变电管理二处专用</string>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>复制</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>textEdit_os</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>textEdit_cpu</zorder>
  <zorder>label_3</zorder>
  <zorder>textEdit_mem</zorder>
  <zorder>label_4</zorder>
  <zorder>textEdit_ip</zorder>
  <zorder>textEdit_ym</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>textEdit_gate</zorder>
  <zorder>textEdit_mac</zorder>
  <zorder>label_7</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_1</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_copall</zorder>
  <zorder>pushButton_quit</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_quit</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_1</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_ip</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_1</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_ip</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_ym</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_ym</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_gate</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_gate</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_mac</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_mac</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_mem</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_mem</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_cpu</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_cpu</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_os</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_os</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
